.tflint.hcl

```diff
@@ -0,0 +1,28 @@
+plugin "terraform" {
+    enabled = true
+    version = "0.2.1"
+    source  = "github.com/terraform-linters/tflint-ruleset-terraform"
+}
+
+rule "terraform_standard_module_structure" {
+  enabled = false
```

Comment:
**discussion required:** Coder is popularising a convention of _everything_ in `main.tf` when it comes to templates. Whereas Hashicorp recommends a particular structure where variables are split into a `variables.tf`. See https://developer.hashicorp.com/terraform/language/modules/develop/structure

question: shall re-enable this rule here and instead refactor our templates to follow the convention?

Reply:
been thinking about this more.  By splitting into seperate files makes it harder for people to gist up and share examples. Conflicted. 😓